dandi/cli/cmd_ls.py

```diff
@@ -4,6 +4,8 @@
 
 import click
 
+from _collections import defaultdict
```

Comment:
`_collections` is an implementation detail.  You should import from `collections` instead.

Reply:
I could have sworn to type `collections`!  not sure how that `_` came about.  Force pushed the fix. Thanks!!